src/clinvar_table_to_vcf.py

```diff
@@ -127,6 +138,7 @@ def table_to_vcf(input_table_path, input_reference_genome, version):
 
         print("\t".join(map(str, vcf_row)))
 
+    sys.stderr.write("Filtered long alleles: Total variants skipped = {}\n".format(long_allele_filtered_count))
```

Comment:
Could you update this so that it includes a reason, please? 

e.g. `Total variants skipped due to index too large to be loaded in the DB`

Reply:
text updated